commands/announce.py

```diff
@@ -207,6 +208,28 @@ def lift_lockdown(self, event, channels):
                         break
         event.msg.reply("Unlock command has successfully completed!")
 
+    @Plugin.command('verification', '<level:str> [reason:str...]')
+    @command_wrapper()
+    def change_verification_level(self, event, level, reason=None):
+        vl = VerificationLevel.get(level.lower())
+        if vl is not None:
+            if event.guild.verification_level != vl:
+                try:
+                    self.bot.client.api.guilds_modify(event.guild.id, reason, verification_level=vl.value)
+                except APIException:
+                    event.msg.reply("Failed to change the server verification level")
```

Comment:
could you parse the exception into GlobalHandlers.handle_exception ? this will make do a data dump in the logs about what exactly went wrong, shouldn't ever happen but in case things do go wrong it's always nice to have logs